src/cpr_data_access/models.py

```diff
@@ -537,6 +540,10 @@ class CPRDocumentWithURL(CPRDocument):
     document_url: Optional[AnyHttpUrl]
 
 
+class DataNotLoadedException(Exception):
+    pass
+
+
```

Comment:
this appears not to be used?

Reply:
Ah yes, decided it was unnecessary so remove dit.